options.go

```diff
@@ -29,9 +28,7 @@ type Options struct {
 	// Env is the "environment" in which the App is running. Default is "development".
 	Env string `json:"env"`
 
-	// LogLevel defaults to "debug". Deprecated use LogLvl instead
-	LogLevel string `json:"log_level"`
```

Comment:
deprecated 4 years ago and now dropped! congratulations, but I think the release note should have a comment on this dropping.